backporter.jl

```diff
@@ -17,7 +17,9 @@ import HTTP
 # Settings #
 ############
 
-BACKPORT = "1.9"
+_BACKPORT_DEFAULT = "1.9"
```

Comment:
I don't think preceeding _'s are needed. This is just a script

Reply:
Done!